scripts/create-component/plop-templates/src/components/{{componentName}}/use{{componentName}}Styles.ts.hbs

```diff
@@ -1,6 +1,8 @@
 import { makeStyles, mergeClasses } from '@griffel/react';
 import type { {{componentName}}State } from './{{componentName}}.types';
 
+
+export const {{componentNames.propertyName}}ClassName = 'fui-{{componentName}}'
```

Comment:
handlebars has build in `camelCase` helper which doesn't work. dropped in nx helpers. see https://github.com/helpers/handlebars-helpers/issues/363